source/jormungandr/jormungandr/instance.py

```diff
@@ -169,8 +177,8 @@ def get_models(self):
     def __repr__(self):
         return 'instance.{}'.format(self.name)
 
-    @memory_cache.memoize(app.config['MEMORY_CACHE_CONFIGURATION'].get('TIMEOUT_PARAMS', 30))
-    @cache.memoize(app.config['CACHE_CONFIGURATION'].get('TIMEOUT_PARAMS', 300))
+    @memory_cache.memoize(app.config[str('MEMORY_CACHE_CONFIGURATION')].get(str('TIMEOUT_PARAMS'), 30))
```

Comment:
So it's tracked by mypy "natively" that flask config is using str?
If so: :tada: great !

Reply:
Since the last version of mypy there is [stubs](https://mypy.readthedocs.io/en/latest/stubs.html) for flask.